chainercv/experimental/links/model/fcis/fcis_resnet101.py

```diff
@@ -0,0 +1,234 @@
+from __future__ import division
+
+import chainer
+import chainer.functions as F
+import chainer.links as L
+import numpy as np
+
+from chainercv.experimental.links.model.fcis import FCIS
+from chainercv.functions import psroi_pooling_2d
+from chainercv.links import Conv2DBNActiv
+from chainercv.links.model.faster_rcnn.region_proposal_network import \
+    RegionProposalNetwork
+from chainercv.links.model.faster_rcnn.utils.loc2bbox import loc2bbox
+from chainercv.links.model.resnet.resblock import ResBlock
+from chainercv.utils import download_model
+
+
+class FCISResNet101(FCIS):
+
+    _models = {
+        'sbd': {
+            'n_fg_class': 20,
+            'url': 'https://github.com/yuyu2172/share-weights/releases/'
+            'download/0.0.6/fcis_resnet101_2018_04_14.npz'
```

Comment:
Could you name the weight in the following protocol?
`fcis_resnet101_DATASETNAME_trained_DATE.npz`?
(e.g., `faster_rcnn_vgg16_voc07_trained_2017_08_06.npz`).

Reply:
OK, i will update